src/components/DownloadCSV.tsx

```
@@ -0,0 +1,104 @@
+    <Box>
+      <Menu>
+        {({ isOpen }) => (
+          <>
+            <MenuButton
+              {...({
+                as: Button,
+                rightIcon: <ChevronDownIcon />,
+                colorScheme: "blue",
+                width: "200px",
+                isActive: isOpen,
+              } as any)}
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)